frontend/app/services/SalesforceService.scala

```diff
@@ -40,6 +40,8 @@ class SalesforceService(salesforceConfig: SalesforceConfig) extends api.Salesfor
   override def updateMemberStatus(user: IdMinimalUser, tier: Tier, customer: Option[Customer]): Future[ContactId] =
     upsert(user.id, memberData(tier, customer))
 
+  override def isAuthenticated = repository.salesforce.authAgent.get().isDefined
```

Comment:
Probably best to shift this implementation-hiding value into the `repository.salesforce` object, which lives inside membership-common, so we're not reaching across repo boundaries.